themes/digital.gov/src/scss/new/_uswds-overrides.scss

```diff
@@ -56,3 +56,13 @@
     mask-image: url("../img/svg-icons/material-icon-inventory_2.svg");
   }
 }
+
+// overrides usa footer until 3.8.0 for uswds comes out and refactor footer into the uswds footer component.
+// turing off sylelint due to it flagging needing to be in BEM format
+/* stylelint-disable */
+@media (min-width: 30em) {
+  .usa-footer .grid-row .mobile-lg\:grid-col-6 {
+    width: 25%;
```

Comment:
Let's use the mixin instead as @mejiaj suggested.

Reply:
@nick-mon1 can you point me to where i can find that mixin for width in percentage?  i was not able to find it in the repo or uswds

Reply:
For percentages we have to use plain old css. The mix-in I meant for media query on line 64.